fvm/environment/programs_test.go

```diff
@@ -122,58 +138,60 @@ func Test_Programs(t *testing.T) {
 
 	var contractASnapshot *state.ExecutionSnapshot
 	var contractBSnapshot *state.ExecutionSnapshot
-	var txAView *delta.View = nil
+	var txASnapshot *state.ExecutionSnapshot
 
 	t.Run("contracts can be updated", func(t *testing.T) {
-		retrievedContractA, err := accounts.GetContract("A", addressA)
+		retrievedContractA, err := getTestContract(
+			mainSnapshot,
+			contractALocation)
 		require.NoError(t, err)
 		require.Empty(t, retrievedContractA)
 
 		// deploy contract A0
-		procContractA0 := fvm.Transaction(
-			contractDeployTx("A", contractA0Code, addressA),
-			derivedBlockData.NextTxIndexForTestingOnly())
-		err = vm.Run(context, procContractA0, mainView)
+		executionSnapshot, output, err := vm.RunV2(
```

Comment:
linter has a point here about output not being used.

Reply:
thx.  i deleted the line that checked the output.  it should be an output.Err should not be nil